README.md

```diff
@@ -83,21 +83,25 @@ You can set up your own deployment by following these steps:
 
 In order to execute these tests, you will need to have a valid test user access token for the Facebook app. You can get an access token by navigating to the [Facebook app portal](https://developers.facebook.com/apps/1650628351692070/roles/test-users/) and clicking "Edit" > "Get an access token for this test user".
 
+10. __Running with SendBird:__ In order to run the SendBird APIs, you will need to configure a SendBird application (or ask other devs for an existing one). You will then need to execute the server with an environment variable containing the SendBird API key. 
+
+To do this on Windows, you can run:  `$env:sendbirdkey="<SECRET>"; node server.js`
+
 
 # API Surface
 
 ## Mobile App API Surface
 
-__Endpoint__: `/token/:fedToken/:authType`
+__Endpoint__: `/register/<facebook||google>?access_token=<token>`
 
 Upgrades a federated identity access token (example: a Facebook access token) for a Hey Mentor identity token, which can be used to authenticate to the API.
 
 Additionally, the identity token is used by the mobile application to determine app behavior, such as whether to present the "Mentor" or "Mentee" UI flow.
 
-Response: Identity Token, or Error
+Response: Identity Token, User ID, or Error
```

Comment:
Is this response doc still accurate, or did we change the return data?

Reply:
Updated the README to remove any old or deprecated info, as part of the travis task I'll add more detail to the setup part and make it easier for new devs to ramp up